### PR TITLE
rowexec: temporarily skip failing test

### PR DESCRIPTION
**Pull request by yuzefovich**
Addresses: #50299.

Release note: None

### CODE REVIEW

**Review by otan — APPROVED** [review]

File: pkg/sql/rowexec/processors_test.go [hunk 1]
```diff
@@ -488,6 +488,7 @@ func TestProcessorBaseContext(t *testing.T) {
 // interesting to test is the integration between DistSQL and KV.
 func TestDrainingProcessorSwallowsUncertaintyError(t *testing.T) {
 	defer leaktest.AfterTest(t)()
+	t.Skip("TODO(yuzefovich): unskip when #50299 is resolved")
```
Comment (otan) [inline pkg/sql/rowexec/processors_test.go:491]: (no way to make this only skip for vectorize?)

**Comment by yuzefovich** [comment]
TFTR!

bors r+

**Review by yuzefovich — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/50308)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @otan)

___
*[pkg/sql/rowexec/processors_test.go, line 491 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/50308#-M9ywDaF5tbpLAKwzzu1-r1-491:-M9z5O2k9wHiJtY8xlS9:b-cp7k7k) ([raw file](https://github.com/cockroachdb/cockroach/blob/2633f774124c81ff8dc1ce4d942e638904b11979/pkg/sql/rowexec/processors_test.go#L491)):*
<details><summary><i>Previously, otan (Oliver Tan) wrote…</i></summary><blockquote>

(no way to make this only skip for vectorize?)
</blockquote></details>

Done. It is a bigger diff than skipping the whole test, so I didn't want to do it initially. PTAL.

**Review by otan — APPROVED** [review]
ah thought you'd just need a t.Skip in the `RunTrueAndFalse` example. this LGTM too.

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/50308#-:-M9zCLOo3-ww4zYD3f0r:bu3q9r8)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @otan)

**Comment by otan** [comment]
___
*[pkg/sql/rowexec/processors_test.go, line 626 at r2](https://reviewable.io/reviews/cockroachdb/cockroach/50308#-M9zChka2IICctJut_NR:-M9zChka2IICctJut_NS:b6odwf2) ([raw file](https://github.com/cockroachdb/cockroach/blob/82538176c897591e8241fa258895e42b9a3226a4/pkg/sql/rowexec/processors_test.go#L626)):*
> ```Go
> 			}
> 			vectorizeMode := "off"
> 			if vectorize {
> ```

actually, i think you can just do
```
if vectorize {
   t.Skip("broken")
}
```
but now I'm just nitpicking. Let's get this in for now!

**Review by yuzefovich — COMMENTED** [review]
TFTR!

bors r+

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/50308#-:-M9zDfT70p27dc_J-B_U:b-oduykh)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @otan)

___
*[pkg/sql/rowexec/processors_test.go, line 626 at r2](https://reviewable.io/reviews/cockroachdb/cockroach/50308#-M9zChka2IICctJut_NR:-M9zDZXW8q73K3znfqHW:b-iatkip) ([raw file](https://github.com/cockroachdb/cockroach/blob/82538176c897591e8241fa258895e42b9a3226a4/pkg/sql/rowexec/processors_test.go#L626)):*
<details><summary><i>Previously, otan (Oliver Tan) wrote…</i></summary><blockquote>

actually, i think you can just do
```
if vectorize {
   t.Skip("broken")
}
```
but now I'm just nitpicking. Let's get this in for now!
</blockquote></details>

You're right. For some reason I didn't realize we have `t` variable easily available in here. Done.

**Pull request merged**